### PR TITLE
Clarify the example UI is generated by random-example.

### PR DESCRIPTION
**Pull request by gyliu513**
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/katib/333)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -83,7 +83,8 @@ Please see [MinikubeDemo.md](./examples/MinikubeDemo.md) for more details.
 ## Web UI
 
 Katib provides a Web UI.
-You can visualize general trend of Hyper parameter space and each training history.
+You can visualize general trend of Hyper parameter space and each training history. The following example UI
+is generated by [random-example](https://github.com/kubeflow/katib/blob/master/examples/random-example.yaml).
```
Comment (hougangliu) [inline README.md]: Thanks! But the example UI is not generated by the example (studyjob name and requestcount don't match at least).

Comment (gyliu513) [reply README.md]: @hougangliu Updated a bit to tell end user can use the example to generate a similar UI. ;-)

**Comment by YujiOshima** [comment]
Thanks!! /approve

**Comment by gyliu513** [comment]
@YujiOshima I think the approve label need in a separate line to take effect ;-)

**Pull request merged**